### ISSUE TITLE
Add'l details screen reader output screen reader output text area

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2923]
![Screenshot 2024-10-08 at 3 41 33 PM](https://github.com/user-attachments/assets/92479a3a-aed3-4b74-9be3-196818a8dbee)
### The ANDI output is:
Anything else? * You can enter up to 2000 characters required

### It should be:

`<legend>`: Is there anything else you’d like us to know about your domain request?
`<label>`: Provide details below. 
`aria-describedby`: You can enter up to 2000 characters
`required`: required

**All together**: Is there anything else you’d like us to know about your domain request? Provide details below. You can enter up to 2000 characters required

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standard that applies:** 
1.3.1 - Info and Relationships (Critical, partially supported)
3.3.2 - Labels or Instructions (Critical, partially supported)
4.1.2 - Name, Role, Value (Critical, partially supported)
**Critical Failure if Not Met?:**  No
**Prioritization:** High/Must Do
**Reason:** This issue impacts accessibility by providing slightly incorrect labeling and descriptions for screen reader users. However given the context of this issue, the screenreader does say "required" in the ARIA output so this is not a complete failure. Ensuring accessible and descriptive labels is essential for clear form navigation, especially for users relying on assistive technologies.
